### PR TITLE
[processing] Fix errors and improve error handling 

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Relate #754

### CODE REVIEW

File: thinkhazard/processing/downloading.py [hunk 1]
```diff
@@ -44,12 +44,24 @@ def argument_parser():
...
+        for filename in os.listdir(cache_path):
+            file_path = os.path.join(cache_path, filename)
+            if os.path.isfile(file_path):
+                os.unlink(file_path)
+
+    def do_execute(self, hazardset_id=None, clear_cache=False):
+        if self.force or clear_cache:
             try:
-                logger.info('Reset all layer to not downloaded state.')
+                logger.info('Resetting all layer to not downloaded state.')
```
Comment (fvanderbiest) [inline thinkhazard/processing/downloading.py]: all layers

File: alembic/versions/7f0c2b165740_add_column_hazardset_processed_error.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+down_revision = '2a26a7ae26a0'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade(engine_name):
+    op.add_column('hazardset', sa.Column('processed_error', sa.String(), nullable=True), schema='processing')
```
Comment (fvanderbiest) [inline alembic/versions/7f0c2b165740_add_column_hazardset_processed_error.py]: I would rename `processed_error` into `processing_error`

File: thinkhazard/processing/completing.py [hunk 3]
```diff
@@ -75,7 +80,8 @@ def do_execute(self, hazardset_id=None):
                 transaction.commit()
             except Exception:
                 transaction.abort()
-                logger.error(traceback.format_exc())
+                logger.error('An error occurred with {}'.format(id),
```
Comment (fvanderbiest) [inline thinkhazard/processing/completing.py]: An error occurred with hazardset {}

**Comment by arnaud-morvan** [comment]
@fvanderbiest : I've fixed tests and added some more tests in test_completing (last commit), if you can take a look.

**Pull request merged**